app/views/CalculationAdditionalView.scala.html

```diff
@@ -36,7 +36,7 @@
 @layout(pageTitle = messages("calculation.headingAndTitle.additional"), isPageFullWidth = false, isUserAgent = user.isAgent,
     extraBreadcrumbs = Some(Seq((messages("index.breadcrumb.index"), controllers.routes.IndexController.onPageLoad.url)))) {
      <header class="hmrc-page-heading govuk-!-margin-top-0">
-         <h1 class="govuk-heading-l govuk-!-margin-bottom-5">@messages("calculation.headingAndTitle.additional")</h1>
```

Comment:
Could you remove `calculation.headingAndTitle.additional = Additional penalty` from the messages file